data/json/items/comestibles/carnivore.json

```diff
@@ -121,6 +121,82 @@
     "flags": [ "SMOKABLE", "RAW" ],
     "smoking_result": "meat_smoked"
   },
+  {
+    "id": "porkbelly",
+    "copy-from": "flesh",
+    "type": "COMESTIBLE",
+    "name": { "str": "chunk of porkbelly", "str_pl": "chunks of porkbelly" },
+    "description": "Fatty cut of pork from the belly, popular in some cuisine.  With proper technique, can be used to create bacon.  Not safe to consume in its current raw form.",
+    "weight": "296 g",
+    "volume": "250 ml",
+    "price": 650,
+    "price_postapoc": 150,
+    "spoils_in": "1 day",
+    "calories": 1300,
+    "healthy": -1,
+    "fun": -10,
+    "vitamins": [ [ "vitC", 0 ], [ "calcium", 0 ], [ "iron", 0 ], [ "vitB", 0 ] ],
+    "flags": [ "SMOKABLE", "RAW" ],
+    "smoking_result": "meat_smoked"
+  },
+  {
+    "id": "raw_cured_fatty_meat",
+    "type": "COMESTIBLE",
+    "copy-from": "flesh",
+    "name": { "str": "slab of raw cured porkbelly", "str_pl": "slabs of raw cured porkbelly" },
+    "description": "Porkbelly that has been preserved and flavored with salt & sugar in order to withdraw the moisture from the first layer.  Still raw, it is ready for smoking into bacon.",
+    "weight": "250 g",
+    "volume": "250ml",
+    "calories": 975,
+    "price": 650,
+    "price_postapoc": 300,
+    "spoils_in": "14 days",
+    "healthy": -5,
+    "vitamins": [ [ "vitC", 0 ], [ "calcium", 0 ], [ "iron", 0 ] ],
+    "flags": [ "SMOKABLE", "RAW" ],
+    "smoking_result": "bacon_uncut"
+  },
+  {
+    "id": "bacon_uncut",
+    "type": "COMESTIBLE",
+    "copy-from": "meat_cooked",
+    "name": { "str": "large slab of uncut bacon", "str_pl": "large slabs of uncut bacon" },
+    "description": "A huge slab of uncut smoked bacon. You can eat as is, but it would be more efficient to cut it into smaller portions with a knife.",
```

Comment:
```suggestion
    "description": "A huge slab of uncut smoked bacon.  You can eat as is, but it would be more efficient to cut it into smaller portions with a knife.",
```